app/models/firm.rb

```diff
@@ -173,6 +174,10 @@ def primary_advice_method
     infer_primary_advice_method
   end
 
+  def main_office
+    offices.first
```

Comment:
I think I know what @Tr4pSt3R means.

There's no special meaning to "main office". The consumer facing side still assumes that there each firm has a single office, and that won't change in the short term. So we have to support just displaying one office, consistently.

We discussed this with the PM, and it's fine for  this _main office_ to simply be the first office added.